src/AppInstallerCommonCore/GroupPolicy.cpp

```diff
@@ -196,6 +196,19 @@ namespace AppInstaller::Settings
                 return std::nullopt;
             }
 
+#ifndef AICLI_DISABLE_TEST_HOOKS

```

Comment:
>AICLI_DISABLE_TEST_HOOKS [](http://example.com/codeflow?start=8&length=24)

Are we planning to expose this in the future?

Reply:
Possibly, but I want that to be an explicit decision, not part of this change.